homeassistant/components/conversation/default_agent.py

```diff
@@ -699,7 +705,14 @@ def _get_or_load_intents(
     def _async_handle_area_registry_changed(
         self, event: core.Event[ar.EventAreaRegistryUpdatedData]
     ) -> None:
-        """Clear area area cache when the area registry has changed."""
+        """Clear area list cache when the area registry has changed."""
+        self._slot_lists = None
+
+    @core.callback
+    def _async_handle_floor_registry_changed(
```

Comment:
Any reason we don't just re-use the callback for the events, since it does the same thing ?

Reply:
They take different event types. I assumed mypy would be mad at me :D